library/FILEBROWSER

```diff
@@ -179,7 +185,8 @@ You specify how many versions to keep.")))
                                                              (* \; "Setup")
               (FNS FB.STARTUP FB.MAKERIGIDWINDOW)
               (FNS FB.PRINTFN FB.COPYFN))
-        (COMS                                                (* \; "commands and major subfunctions")
+        (COMS                                                (* \; 
+                                                           "commands and major subfunctions")
```

Comment:
I'd still like to know why this got split onto two lines when, for example, at line 179 above it did not add a line-break.

Reply:
i made a separate issue. Once we've fixed the various problems that make the files ugly when printed, we can re-prettyprint everything.